Assets/HoloToolkit-Tests/Utilities/Scripts/Extensions/Editor/GameObjectExtensionsTests.cs

```diff
@@ -0,0 +1,209 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace HoloToolkit.Unity.Tests
```

Comment:
No worries, But that's where the other integration tests are located at the moment.